pipes/WDL/tasks/tasks_terra.wdl

```diff
@@ -30,6 +30,42 @@ task gcs_copy {
   }
 }
 
+task get_gcloud_env_info {
+  input {
+    String? additional_command_to_run
+  }
+  meta {
+    description: "task for inspection of backend env, and optionally running a command"
+  }
+  parameter_meta {
+    additional_command_to_run: {
+      description: "Command that can optionally be included and executed as part of this task"
+    }
+  }
+  command {
```

Comment:
```suggestion
  command <<<
```

Reply:
probably need to switch syntax?

Reply:
Are we standardizing toward `<<<` to [discourage](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#expression-placeholders) WDL interpolation of`${}`?

Reply:
Yes there's definitely a number of bashisms that simply won't work with the older WDL syntax